doc/mk_sdcard_image.sh

```diff
@@ -266,6 +266,7 @@ make_image()
 	print_step "Preparing image at ${img_output_file}"
 
 	sudo umount -f ${MOUNT_POINT} || true
+        ls ${img_output_file}?* | xargs -n1 sudo umount -l -f || true
```

Comment:
Something is wrong with formatting there.

Reply:
Done